src/components/crossDevice/Intro/index.js

```diff
@@ -21,24 +21,27 @@ const Intro = ({translate, nextStep, mobileConfig}) => {
     ),
     'return-computer': 'return_computer',
   }
+  const stageList = Object.keys(stages)
```

Comment:
Also here - not sure if you need to set a separate constant when it's used only in one place.